### ISSUE TITLE
How to generate the assets dist directory

### ISSUE THREAD

**Issue by mvargasmoran** [htmlburger/carbon-fields#523]
Hello, can you help me compile the assets as the version (this one) that you distribute with composer doesn't provide them as opposed to the Wordpress Plugin.

If you tell me how, I'll contribute the PR for the documentation.

Thanks in advance.

**Comment by dmhendricks** [comment]
Are you loading a release version (currently 2.2.0) or the master branch? If you load the release version, it includes them.

That said, I'd be curious to the answer as well, since sometimes I used the development branch when there is a bug fix that I need. I've been having to copy it manually.

**Comment by mvargasmoran** [comment]
TL;DR;  **Thanks! it works**

Both of your answers work, the `npm run build` works well but I don't want to have to add that to my deploy.

@dmhendricks I am using your own wordpress-base-plugin, I resolved the issue by deleting the dependency and running:
`composer require htmlburger/carbon-fields`
which now installed 2.2.0 as you say, but when I came across your base plugin and followed your documentation ["as a composer dependency"](https://github.com/dmhendricks/wordpress-base-plugin/wiki#composer-dependency) it installed `"htmlburger/carbon-fields": "dev-master"` which didn't had the `dist` directory in it.

Well thanks again for your answer and for your base plugin, I'm even using it as a guide to update some messy legacy plugins, you've been of great help.
